http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -53,14 +78,81 @@ class GKEInterpreter[F[_]: Parallel: ContextShift](
       getApp <- F.fromOption(getAppOpt, AppNotFoundException(msg.project, msg.appName, ctx.traceId))
       gkeClusterId = getApp.cluster.getGkeClusterId
       namespaceName = getApp.app.appResources.namespace.name
+      googleProject = getApp.cluster.googleProject
```

Comment:
Might need to handle configs once https://github.com/DataBiosphere/leonardo/pull/1565 is merged (and tests are fixed).. I changegd this method to take `params: CreateAppParams` which includes `googleProject`.

Reply:
i imagine i will have lots of merge conflicts hehe but i'll make sure you merge first since your PR is much bigger and dealing with merge conflicts would be more annoying!